kitchen.yml

```diff
@@ -0,0 +1,209 @@
+  - name: fedora-31-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:fedora-31
+  - name: opensuse-leap-151-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:opensuse-leap-15.1
+      run_command: /usr/lib/systemd/systemd
+    # Workaround to avoid intermittent failures on `opensuse-leap-15.1`:
+    # => SCP did not finish successfully (255):  (Net::SCP::Error)
+    transport:
+      max_ssh_sessions: 1
+  - name: amazonlinux-2-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:amazonlinux-2
+  - name: ubuntu-1804-2019-2-py2
+    driver:
+      image: netmanagers/salt-2019.2-py2:ubuntu-18.04
+  - name: amazonlinux-1-2019-2-py2
+    driver:
+      image: netmanagers/salt-2019.2-py2:amazonlinux-1
+      run_command: /sbin/init
+  - name: arch-base-latest-2019-2-py2
+    driver:
+      image: netmanagers/salt-2019.2-py2:arch-base-latest
+      run_command: /usr/lib/systemd/systemd
+
+  ## SALT `2018.3`
+  - name: centos-7-2018-3-py3
+    driver:
+      image: netmanagers/salt-2018.3-py3:centos-7
+  - name: fedora-30-2018-3-py3
+    driver:
+      image: netmanagers/salt-2018.3-py3:fedora-30
+  - name: debian-9-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:debian-9
+  - name: debian-8-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:debian-8
+  - name: ubuntu-1804-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:ubuntu-18.04
+  - name: ubuntu-1604-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:ubuntu-16.04
+  - name: centos-6-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:centos-6
+      run_command: /sbin/init
+  - name: opensuse-leap-151-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:opensuse-leap-15.1
+      run_command: /usr/lib/systemd/systemd
+    # Workaround to avoid intermittent failures on `opensuse-leap-15.1`:
+    # => SCP did not finish successfully (255):  (Net::SCP::Error)
+    transport:
+      max_ssh_sessions: 1
+  - name: amazonlinux-1-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:amazonlinux-1
+      run_command: /sbin/init
+  - name: arch-base-latest-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:arch-base-latest
+      run_command: /usr/lib/systemd/systemd
+
+provisioner:
+  name: salt_solo
+  # log_level: debug
+  salt_install: none
+  require_chef: false
+  formula: pycharm
+  salt_copy_filter:
+    - .kitchen
+    - .git
+
+verifier:
+  # https://www.inspec.io/
+  name: inspec
+  sudo: true
+  # cli, documentation, html, progress, json, json-min, json-rspec, junit
+  reporter:
+    - cli
+
+suites:
+  - name: default
+    provisioner:
+      state_top:
+        base:
+          '*':
+            - pycharm.clean
+            - pycharm
+      pillars:
+        top.sls:
+          base:
+            '*':
+              - pycharm
+      pillars_from_files:
+        pycharm.sls: test/salt/default/pillar.sls
```

Comment:
The general standard we decided upon for test pillars is:

* `test/salt/pillar/<suite-name-or-similar>.sls`
  - We've not been too picky about the name of the actual file, as long as it's clear
  - Personally, I believe it should always contain the InSpec suite name at the very least, if that's not already sufficient

You can see how this is being used across the org by searching for `pillars_from_files` in the `formulas.yaml` of the `ssf-formula`:

* https://github.com/myii/ssf-formula/blob/master/ssf/formulas.yaml

In this PR, that would also affect the line further below.